bin/boring-tsc.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+const fs_extra = require('fs-extra');
+const path = require('path')
+const paths = require('../dist/node_modules/paths')
+const child_process = require('child_process')
+const fs = require('fs');
+
+
+async function run() {
+
+  let tscPath =  '/../node_modules/.bin/tsc';
+  try {
+    fs.statSync(path.normalize(__dirname+tscPath));
+  } catch(e) {    
```

Comment:
okay what hilarious drama makes this necessary?

Reply:
Also why do you need to stat these files if you're not doing anything with the return value? stat shouldn't have side effects... right??

Reply:
`stat` is to just to make sure we know the valid path.  The only reason why it throws into the catch is when you are doing local development have have npm'd linked the an application to boring where both repos live in the same parent directory.